src/gsi/m_berror_stats.f90

```diff
@@ -98,10 +109,13 @@ module m_berror_stats
 !
 ! !INTERFACE:
 
+!!subroutine get_dims(mype,msig,mlat,mlon,lunit)
 subroutine get_dims(msig,mlat,mlon,lunit)
 
+   use mpimod, only: mype
    implicit none
 
+!!   integer(i_kind)         ,intent(   in) :: mype  ! proc identifier
```

Comment:
if we get this from mpimod, we don't need it defined here

Reply:
Remove commented out line at 68d52ac